Resources/Prototypes/Recipes/Lathes/devices.yml

```diff
@@ -19,3 +19,16 @@
   materials:
     Steel: 200
     Plastic: 300
+
+- type: latheRecipe
+  id: FlashPayload
+  icon:
+    sprite: Objects/Devices/payload.rsi
+    state: payload-flash-armed
+  result: FlashPayload
+  completetime: 1
+  materials:
+    Steel: 25
+    Plastic: 50
+    Glass: 25
+    #one fourth of what making a flash would cost
```

Comment:
this should be quite a bit more expensive imo, shouldnt be this easy to essentially mass manufacture flashbangs

Reply:
sure , increase it by 2 fold ? I maded it a fourth of a chem payload.